proposals/notifications.md

```diff
@@ -0,0 +1,88 @@
+# Solid Notifications
+
+Solid notifications are intended to announce social interactions and data. A Solid notification is an [LDP Resource](https://www.w3.org/TR/ldp/#ldpr) which is submitted to and resides in an [LDP Container](https://www.w3.org/TR/ldp/#ldpc), i.e., a [Solid inbox](inbox.md).
+
+The contents of a notification is application specific. We derive two core principles from this:
+
+1. a notification can contain arbitrary data
+2. a notification is vocabulary agnostic
+
+Applications can create notifications to use any vocabulary to describe its contents in any shape and form. Multiple notifications can be contained inside a single resource or span across different resources. A single resource with multiple notifications may be suitable for `HTTP PATCH`, whereas multiple resources containing a single notification may be suitable for `HTTP POST` or `PUT`.
+
+
+## Discovery
+As notifications are contained in inboxes, a Solid inbox container has `ldp:contains` relations to each notification resource. If a [Solid outbox](outbox.md) is employed, it may also contain references to the notification.
+
+
+## Notification integrity
+Adhering to the integrity rules of an inbox notification e.g., for the purpose of a verification criteria or discovery is further described in [Solid inbox](inbox.md#shapes-constraint), is the responsibility of applications which submit and verify them.
```

Comment:
"Shapes constraint` is derived from [SHACL](https://www.w3.org/TR/shacl/) but it could be any mechanism which could indicate the _shape_ and _form_ of the notification e.g., the RDF graph. It essentially tells the client "Hey, if you want to submit a valid or useful notification to this inbox, it should contain this". This is opens up a set of applications which can consume inbox notifications and decide what to filter, or how to display etc.

I'll expand on that in inbox.md.